daemon/logger/awslogs/cloudwatchlogs.go

```diff
@@ -168,11 +168,8 @@ func New(info logger.Info) (logger.Logger, error) {
 		go func() {
 			backoff := 1
 			maxBackoff := 32
-			for {
+			for !containerStream.closed.Load() {
 				// If logger is closed we are done
```

Comment:
This comment looks a bit odd now; probably should be slightly rewritten (could be outside the loop, or after the `for` line, and describe continue reading until it's closed). Or perhaps just removing it would be fine.